movie-theater/src/main/java/com/epam/spring/core/movietheater/service/AuditoriumService.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.spring.core.movietheater.service;
+
+import java.util.List;
+
+import com.epam.spring.core.movietheater.dao.AuditoriumDao;
+import com.epam.spring.core.movietheater.model.Auditorium;
+
+public class AuditoriumService {
+	private AuditoriumDao auditoriumDao;
+	
+	public AuditoriumService(AuditoriumDao auditoriumDao) {
+		this.auditoriumDao = auditoriumDao;
+	}
+	public Auditorium getByName(String name) {
+		return auditoriumDao.getByName(name);
+	}
+	public List<Auditorium> getAll(){
+		return auditoriumDao.getAll();
+	}
+}
```

Comment:
Please use "coding to interface" principal. Best practice is to have interface **AuditoriumService** and than class **AuditoriumServiceImpl implements AuditoriumService**. You should create folder **impl** in folder **service** and all your ...Impl classes should be there

Reply:
project updated according ti this principal